vnext/Microsoft.ReactNative/Fabric/Composition/CompositionEventHandler.cpp

```diff
@@ -977,6 +979,11 @@ void CompositionEventHandler::getTargetPointerArgs(
   tag = -1;
 
   ptScaled = {position.X, position.Y};
+
+  // In the case of a sub rootview, we may have a non-zero origin.  hitTest takes a pt in the parent coords, so we need
+  // to apply the current origin
+  ptScaled += RootComponentView().layoutMetrics().frame.origin;
```

Comment:
comment says the hitTest pt is taken in parent coordinates and we're incorporating the origin in the subrootview (separate window) for it hit test... for the other lines I was thinking "should this origin be incorporated inside hitTest instead of called before it, if it's the RootComponentView()'s layoutMetrics?" ... then saw the `if(capturedPointers) { ... }` that's more curious.  When do we have `m_capturedPointers` vs not?

Is the underlying type of `facebook::react::Point` a float?  Considering the implications of different DPI on different displays which can do curious things in some contexts